.nvmrc

```diff
@@ -0,0 +1 @@
+10.15.1
```

Comment:
I always have problems when the node version isn't specified

Reply:
Hm I am unfamiliar with engines! But I think it's probably pretty fair to decide that we use nvm? Any reason that would be unreasonable?